pkg/components/components.go

```diff
@@ -657,12 +657,15 @@ func GetClusterRole() *rbacv1.ClusterRole {
 	}
 
 	role.Rules = append(role.Rules, componentsClusterRoles()...)
+	if allowMultus {
+		role.Rules = append(role.Rules, multusClusterRoles()...)
+	}
```

Comment:
nit: would be nice that it will be before line 659
this way it is more encapsulated by groups
as multus is closer to its real place where it belongs

even one blank line in between can be added additionally

wdyt ?